sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md

```diff
@@ -3,9 +3,11 @@
 ## 4.6.0 (2021-08-10)
 
 ### Features Added
-
 - Added `ContainerServiceNewKubernetesVersionAvailableEventData` system event.
 
+### Key Bugs Fixed
```

Comment:
```suggestion
### Bugs Fixed
```

I believe they changed this;  only relevant if you'd like it to roll into the release notes.